src/data/pages.json

```diff
@@ -536,6 +550,28 @@
                     "id": "retest-runtime-permission-grants"
                   }
                 ]
+              },
+              {
+                "title": "Known issues",
+                "url": "/documentation/develop/known-issues/",
+                "subpageitems": [
+                  {
+                    "title": "Content scripts don't appear in DevTools",
+                    "id": "invisible-content-scripts"
+                  },
+                  {
+                    "title": "Extension source don't update in DevTools",
+                    "id": "devtools-updates-missing"
+                  },
+                  {
+                    "title": "\"Destroyed actor\" errors when debugging Android",
+                    "id": "android-destroyed-actor"
+                  },
+                  {
+                    "title": "Empty temporary extensions list when debugging Android",
+                    "id": "android-temp-ext-missing"
```

Comment:
I don't think this point at a section in the article anymore
```suggestion
```

Reply:
You're right, I forgot to update this doc after dropping that section. Thanks for catching this!